### DISCUSSION TITLE
Reset automatic suggestions

### DISCUSSION THREAD

**Discussion by JaCoB1123** [WeblateOrg/weblate#12997] [Q&A] [answered]
Is there a way to reset the automatic suggestions? For example when I'm translating something, create a new item in the glossary beforehand and now I'd like to reevaluate the suggestions.

**Comment by nijel** [comment]
Are you talking about some specific source of the suggestions? Or the ones already added to Weblate as suggestions?

**Reply by JaCoB1123** [reply]
For me it's primarily concerning OpenAI's suggestions.

**Reply by nijel** [reply, marked as answer]
All suggestions from external services are cached. The cache will currently be used even if you change glossary terms (I've created https://github.com/WeblateOrg/weblate/issues/13122 to address this). As a workaround, you can change the service configuration, that will reset the cache as well.

**Reply by chillfun420** [reply]
> Are you talking about some specific source of the suggestions? Or the ones already added to Weblate as suggestions?
